spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -381,7 +381,7 @@ function chanOpenAck(
   proofTry: CommitmentProof,
   proofHeight: Height) {
     channel = provableStore.get(channelPath(portIdentifier, channelIdentifier))
-    abortTransactionUnless(channel.state === INIT || channel.state === TRYOPEN)
+    abortTransactionUnless(channel.state === INIT)
```

Comment:
@AdityaSripal I'm not sure I understand why the crossing hellos scenario is no longer possible.

Reply:
Does this [discussion](https://github.com/cosmos/ibc/pull/629#discussion_r770569775) help?